jvm/ml4ir-inference/src/test/integration/ml4ir/inference/tensorflow/TensorFlowInferenceIT.scala

```diff
@@ -50,18 +86,105 @@ class TensorFlowInferenceIT extends TestData {
    * @return List of PredictionVector
    */
   def readPredictionCSV(csvFile: String): List[PredictionVector] = {
-    return Source.fromFile(csvFile).getLines.toList.tail.flatMap(extractColumnValues)
+    Source.fromFile(csvFile).getLines.toList.tail.flatMap(extractColumnValues)
   }
 
+  def validateRankingScores(query: StringMapQueryAndPredictions,
+                     sequenceExample: SequenceExample,
+                     scores: Array[Float],
+                     numDocs: Int) = {
+    val docScores = scores.take(numDocs)
+    val maskedScores = scores.drop(numDocs)
+    docScores.foreach(
+      score => assertTrue("all docs should score non-negative", score > 0)
+    )
+    for {
+      maskedScore <- maskedScores
+      docScore <- docScores
+    } {
+      assertTrue(
+        s"docScore ($docScore) should be > masked score ($maskedScore)",
+        docScore > maskedScore
+      )
+    }
+    if (query.predictedScores != null) {
+      assertArrayEquals("scores aren't close enough: ", docScores, query.predictedScores, 1e-6f)
+    }
+
+  }
+
+  @Test
+  def testRankingSavedModelBundleWithCSVData(): Unit = {
```

Comment:
@ducouloa Is there a way to run this function by passing say command line arguments to the model and predictions CSV? I ask because I am interested in using this when we train every single model - like a seal of approval and the corresponding jar version associated with it.
CC: @Ullimague

Reply:
One fairly simple thing we could really use, @ducouloa , is the ability to do e.g.:

```
mvn test -Dtest=TensorFlowInferenceIT#testRankingSavedModelBundleWithCSVData -DbundleLocation=/path/to/my/trained/model -DrunName=myRunName
```

This would remove the need for a "standalone bootstrap", right?

Reply:
actually @ducouloa , @lastmansleeping - what I wrote above literally just works.  You just need to have the paths pointing to the right place (i.e. where a) the model bundle is, b) where the python logs are, and c) where the feature_config.yaml is.  If all those are right, the above `mvn test` commandline will work on any ranking model, as long as you ran training in train_inference_evaluate with a feature_config with log_at_inference=true

Reply:
That's great if we can call it from command line. I can hook it up at the end of the Flowsnake model training.

Reply:
@ducouloa if we are not adding the bootstrap, I think we should document the CLI command that @jakemannix shared.

Reply:
i added a note in the main README.md for this.